src/main/resources/view/MainWindow.fxml

```diff
@@ -46,12 +63,11 @@
           </padding>
         </StackPane>
 
-        <VBox fx:id="personList" styleClass="pane-with-border" minWidth="340" prefWidth="340" VBox.vgrow="ALWAYS">
+        <StackPane VBox.vgrow="NEVER" fx:id="commandBoxPlaceholder" styleClass="pane-with-border">
           <padding>
-            <Insets top="10" right="10" bottom="10" left="10" />
+            <Insets top="5" right="10" bottom="5" left="10" />
```

Comment:
Should the left and right inset be 0 instead? It looks better if it fills the whole width. Same issue with the above resultDisplayPlaceholder.

Reply:
Padding removed so everything fills up the width